### PR TITLE
Add options for FIO profile to generate S3 load for noobaa autoscale

### PR DESCRIPTION
**Pull request by tiffanyn108**
Add options for FIO profile to generate S3 load to >80% of CPU utilization.  This will trigger noobaa autoscale creating additional endpoint.
This will fix the issue: https://github.com/red-hat-storage/ocs-ci/issues/3282

Signed-off-by: Tiffany Nguyen <tunguyen@redhat.com>

### CODE REVIEW

File: ocs_ci/templates/workloads/fio/config_s3.fio [hunk 1]
```diff
@@ -32,3 +32,24 @@ size=64k
...
+bs=64k
+size=512m
+numjobs=4
+
+[job3]
+iodepth=32
+rw=randrw
+bs=128k
+size=1024m
+numjobs=4
```
Comment (nb-ohad) [inline ocs_ci/templates/workloads/fio/config_s3.fio:55]: What will happen to all the other code that uses `mcg_job_factory`? will default invocation will ignore all these new jobs?

Comment (tiffanyn108) [reply ocs_ci/templates/workloads/fio/config_s3.fio:55]: if you don't have options specify in the test case, mcg_job_factory will pick up default fio config which doesn't have enough load.  Default config is set: https://github.com/red-hat-storage/ocs-ci/blob/8d44436d54ab29e581a16ce0783a62a2a8601811/ocs_ci/templates/workloads/fio/config_s3.fio#L22

**Comment by tiffanyn108** [comment]
Test case passed on production AWS cluster.

File: ocs_ci/templates/workloads/fio/config_s3.fio (hunk 1 above)
Comment (fbalak) [reply ocs_ci/templates/workloads/fio/config_s3.fio:55]: This is actually the default config file. This way all tests that use `mcg_job_factory` would use this configuration. I suggest to add these options to test in `tests/manage/mcg/test_endpoint_autoscale.py` like:
```
options = {
    'create': [('name', 'job1'), ('name', 'job2'), ('name', 'job3'), ('runtime', '900')],
    'job1':[('iodepth', '4'), ('rw', 'randrw'), ('bs', '32k'), ('size', '64m'), ('numjobs', '4')],
    'job2':[('iodepth', '16'), ('rw', 'randrw'), ('bs', '64k'), ('size', '512m'), ('numjobs', '4')],
    'job3':[('iodepth', '32'), ('rw', 'randrw'), ('bs', '128k'), ('size', '1024m'), ('numjobs', '4')]
}
job = mcg_job_factory(custom_options=options)
```

**Review by wusui — APPROVED** [review]
LGTM

File: ocs_ci/templates/workloads/fio/config_s3.fio (hunk 1 above)
Comment (nb-ohad) [reply ocs_ci/templates/workloads/fio/config_s3.fio:55]: I prefer @fbalak suggestion/solution as it was my original recommendation to do it that way

**Review by Avilir — APPROVED** [review]

**Comment by ebenahar** [comment]
@tiffanyn108 I guess this PR will fix https://github.com/red-hat-storage/ocs-ci/issues/3282, right?

**Comment by tiffanyn108** [comment]
@ebenahar Yes, this is for fixing https://github.com/red-hat-storage/ocs-ci/issues/3282

File: ocs_ci/templates/workloads/fio/config_s3.fio (hunk 1 above)
Comment (tiffanyn108) [reply ocs_ci/templates/workloads/fio/config_s3.fio:55]: @nb-ohad I will sync up with @fbalak on this.

Comment (tiffanyn108) [reply ocs_ci/templates/workloads/fio/config_s3.fio]: Fixed.

File: ocs_ci/ocs/fio_artefacts.py [hunk 2]
```diff
@@ -56,6 +56,8 @@ def get_mcg_conf(mcg_obj, workload_bucket, custom_options=None):
     # add or overwrite custom values
     if custom_options:
         for section in custom_options:
+            if section is not 'create':
```
Comment (shylesh) [inline ocs_ci/ocs/fio_artefacts.py:59]: @tiffanyn108 
You can use ```config.has_section(section)``` function to verify if such a section exists or not then based on that take decision whether to add or set

Comment (tiffanyn108) [reply ocs_ci/ocs/fio_artefacts.py]: Acked, fixed it.

**Review by keesturam — APPROVED** [review]
LGTM

**Review by wusui — APPROVED** [review]
LGTM (I just noticed that I commented on this without approving it earlier)

**Review by fbalak — APPROVED** [review]

**Review by nb-ohad — APPROVED** [review]

**Pull request merged**